Developers/Newcomers_Guide.rst

```diff
@@ -46,7 +46,7 @@ Step 3. Picking Up an Issue
 ---------------------------
```

Comment:
could you wrap the lines at 72 chars in the commit body? else they will be lost as github wraps them automatically :/ thanks